x/fswap/types/msgs_test.go

```diff
@@ -89,49 +92,328 @@ func TestAminoJSON(t *testing.T) {
 func TestQuerySwapRequestValidate(t *testing.T) {
```

Comment:
I think these part is duplicated in `msg_server_test`.
Could we standardize the validation of messages? Except for the swap, the other three messages do not have a validation function. I suggest that QuerySwapRequest could also be validated similarly. And currently it only checks if the value is null without using sdk.ValidateDenom, which I think is insufficient.

Reply:
After I checked duplication, `TestQuerySwapRequestValidate` is duplicated and redundant.
Thank you, I'll remove it

> And currently it only checks if the value is null without using sdk.ValidateDenom, which I think is insufficient.
I agree with it. 
I want to add test cases only and don't want to change production code. We may update insufficient validation logic in other PR.